backend/app/domains/users/crud.py

```
@@ -0,0 +1,87 @@
+            hashed_password = self.get_password_hash(user_data.password)
+            user = User(
+                username=user_data.username,
+                email=user_data.email,
+                hashed_password=hashed_password,
+                role=user_data.role
+            )
+            self.session.add(user)
+            self.session.commit()
+            self.session.refresh(user)
+            return user
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)